src/transformers/dynamic_module_utils.py

```diff
@@ -185,35 +185,20 @@ def check_imports(filename: Union[str, os.PathLike]) -> List[str]:
     return get_relative_imports(filename)
 
 
-def get_class_in_module(repo_id: str, class_name: str, module_path: Union[str, os.PathLike]) -> typing.Type:
+def get_class_in_module(class_name: str, module_path: Union[str, os.PathLike]) -> typing.Type:
```

Comment:
could you return the original parameter `repo_id` also the documentation for it too, changing parameter names or removing them should not be be done this fast, usually the maintainers deperacate a parameter after a big release of the library in case other libraries or modules use this function too, so it's too much effort.

Also tried a variation of this and it looks stable, nice work
pics for reference

![image](https://github.com/huggingface/transformers/assets/70411813/b616362d-91af-434f-bc54-d00b6cc4bfdd)
![image](https://github.com/huggingface/transformers/assets/70411813/cf03a977-13c9-4d2f-acb1-b3dff92a96aa)
![image](https://github.com/huggingface/transformers/assets/70411813/beecbf21-e824-4b78-a0be-6267b1e96316)
![image](https://github.com/huggingface/transformers/assets/70411813/95959d5c-0224-4c28-93e0-0e8c2fa862f1)
![image](https://github.com/huggingface/transformers/assets/70411813/a8b4f22f-ba37-43ee-9cd6-647f722504c9)
and the remote repo is still working perfectly 
![image](https://github.com/huggingface/transformers/assets/70411813/5066c992-ed09-4d8d-a1c4-28c9ce9b6c24)

Reply:
> could you return the original parameter `repo_id` also the documentation for it too

The parameter was only added 10 hours ago in #29175, and there hasn't been a release since then. I don't think many people are already relying on this. And that change was merged without any deprecation period, so it doesn't seem like this is considered a stable public API.

Reply:
@cebtenzzre i just rechecked, and yess it checks out, thanks a lot for clearing that out